cmd/oras/pull.go

```diff
@@ -141,9 +151,14 @@ func runPull(opts pullOptions) error {
 	copyOptions.PostCopy = func(ctx context.Context, desc ocispec.Descriptor) error {
 		name := desc.Annotations[ocispec.AnnotationTitle]
 		if name == "" {
-			return nil
+			if !opts.Verbose {
+				return nil
+			}
+			name = desc.MediaType
+		} else {
+			// named content downloaded
+			pulledEmpty = false
 		}
```

Comment:
It seems that this code block can be replaced with `PrintStatus`. Could you have a check?

Reply:
That's a bit different. Let's keep it.